volatility/framework/layers/crash.py

```diff
@@ -168,8 +176,8 @@ def _load_segments(self) -> None:
                         mapped_offset += length
 
 
-                # Finish it off
-                if (bit_position == (len(summary_header.BufferLong) * 32) - 1 or bit_position == (len(summary_header.BufferLong) * 32) - 1 -32*0x2000) and current_bit == 1:
+                # Find the last segment in a file which will be at the end or two pages from the end
+                if (bit_position == bitmap_len - 1 or bit_position == bitmap_len - 1 -32*0x2000) and current_bit == 1:
```

Comment:
So I think the -32*0x2000 is still a little difficult to figure?  Any chance of a bit more clarification around what's going on here please?